internal/projection/fakevalkey_test.go

```
@@ -0,0 +1,441 @@
+		if err != nil {
+			close(f.done)
+			return
+		}
+		go f.session(conn)
+	}
+}
+
+// session serves one connection: read a command array, execute, write the
+// reply, until either side drops.
+func (f *fakeValkey) session(c net.Conn) {
+	defer c.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `c.Close` is not checked

(errcheck)